modules/tinymce/src/plugins/link/main/ts/core/NormalizeLink.ts

```diff
@@ -0,0 +1,65 @@
+import { Fun, Optional } from '@ephox/katamari';
+import RangeUtils from 'tinymce/core/api/dom/RangeUtils';
+import DomTreeWalker from 'tinymce/core/api/dom/TreeWalker';
+import Editor from 'tinymce/core/api/Editor';
+
+const isBR = (elm: Node): Boolean => elm && elm.nodeName.toLowerCase() === 'br';
```

Comment:
These should all return `boolean` not `Boolean` since that is the object type.

Reply:
updated